app/views/miq_ae_customization/_dialog_field_form_dynamic_options.html.haml

```diff
@@ -28,4 +28,91 @@
     = check_box_tag('field_auto_refresh', '1',
                     @edit[:field_auto_refresh],
                     'data-miq_observe_checkbox' => {:url => url}.to_json)
+.form-group
```

Comment:
@eclarizio i agree, and I have that on my mind to refactor this, so in future we don't have to maintain these fields in 2 places.